src/accordion/accordion.ts

```diff
@@ -32,6 +32,36 @@ export class NgbPanelContent {
 }
 
 /**
+ * This directive must be used to wrap accordion panel header.
+ */
+@Directive({selector: 'template[ngbPanelHeader]'})
+export class NgbPanelHeader {
+  constructor(public templateRef: TemplateRef<any>) {}
+}
+
+/**
+ * The context available to the NgbPanelHeader directive
+ */
+export class NgbPanelHeaderContext implements NgbPanelHeaderContext {
```

Comment:
That's a mistake. There should only be a class. I just forgot to remove the interface. Not even sure why that compiles BTW. I originally started with just a class, then noticed that regular classes without any decorator were not documentetd, so introduced an interface. Then I thought that I should rather improve the documentation process and removed the interface, but forgot this implements clause.